othello/apps/games/tasks.py

```diff
@@ -0,0 +1,201 @@
+import logging
+from datetime import datetime, timedelta
+from time import sleep
+
+from asgiref.sync import async_to_sync
+from celery import shared_task
+from celery.utils.log import get_task_logger
+from channels.layers import get_channel_layer
+
+from django.conf import settings
+from django.db.models import Q
+from django.utils import timezone
+
+from ...moderator import *
+from ...moderator.runners import PlayerRunner, ServerError, UserError, YourselfRunner
+from ..games.models import Game, Player, Submission
+
+logger = logging.getLogger("othello")
+task_logger = get_task_logger(__name__)
+
+
+def send_through_socket(game, event_type):
```

Comment:
Please rename this to something that is actually correct, like `send_through_game_channel()`.